### ISSUE TITLE
Business Key for Call Activity is always checked

### ISSUE THREAD

**Issue by ingorichtsmeier** [viadee/camunda-modeler-tooltip-plugin#5]
First, Thank you for this wonderful and very useful plugin!

I recently found a an issue with the call activities: In the overlay the business key is always checked, even if in the property panel the business key is unchecked:
![business-key-tooltip-plugin_LI](https://user-images.githubusercontent.com/7870843/103799655-a96a0900-504b-11eb-9a8f-9e8efec5b94a.jpg)

It's a bit confusing.

**Comment by rnschk** [comment]
Hi @ingorichtsmeier, thank you very much for your feedback. :) 

I tried to reproduce your issue, but call-activity-elements work correctly in my setting, even if the business-key is checked and unchecked in different situations:
![image](https://user-images.githubusercontent.com/5700718/103832708-3b3f3980-507f-11eb-8d9f-1a6b4a18c276.png)
  
![image](https://user-images.githubusercontent.com/5700718/103832714-41351a80-507f-11eb-9e8e-58f3a84e6b1c.png)

Could you please reveal your modeler's version and maybe any extraordinary setting, or maybe even provide the respective process-model?

BR Florian

**Comment by ingorichtsmeier** [comment]
Hi @rnschk,

here is a process model to reproduce:

[CallActivityToolTip.zip](https://github.com/viadee/camunda-modeler-tooltip-plugin/files/5780428/CallActivityToolTip.zip)

The error appeared after adding a variable input mapping to the first Call Activity. It doesn't propagate the business key.

Kind regards, Ingo

**Comment by rnschk** [comment]
Hi @ingorichtsmeier, thank you. I just released v0.0.6 that should fix this issue.

The issue was caused by the variables-in-mapping used in call-activity 'CA 1'. Looking at the xml-data, one can see that both, the business-key as well as the variable mapping are written as extension-elements of type 'camunda:in', which finally lead to this behaviour. 
```xml
<bpmn:callActivity id="0cfxanm" name="CA 1" calledElement="hello">
  <bpmn:extensionElements>
    <camunda:in source="1" target="1" />
    <camunda:in businessKey="#{execution.processBusinessKey}" />
  </bpmn:extensionElements>
  ...
</bpmn:callActivity>
```

BR Florian
